composer.json

```diff
@@ -22,5 +22,10 @@
         "branch-alias": {
             "dev-master": "1.x-dev"
         }
+    },
+    "config": {
+        "allow-plugins": {
```

Comment:
Just use `"allow-plugins": false`, so we're set for the future if similar shitshows occur again.

Reply:
Updated in https://github.com/netgen/better-ibexa-admin-ui/pull/4/commits/7f33c844e6605041fbc2d4bc0f4b963aecaf4a52.